Assets/sass/main.scss

```diff
@@ -0,0 +1,384 @@
+$primary: #ec5242;
```

Comment:
- [ ] **You are using the same properties multiple times throughout your stylesheets. It will be ideal to extract them in their own classes**

Properties that won't have a dynamic range of values can be extracted in their own classes. For instance, 
```
.d-flex {
  display: flex
}

.text-center {
 align-items: center
}
```

Also, as you are using Bootstrap, it is not very ideal to include explicit declaration for things like `display: flex`, `display: none`, `text-align`, etc. Bootstrap has classes for a maximum number of properties.